tests/unit/Auth/AuthTest.php

```diff
@@ -48,12 +48,121 @@ public function testHash()
 
     public function testPassword()
     {
-        $secret = 'secret';
-        $static = '$2y$08$PDbMtV18J1KOBI9tIYabBuyUwBrtXPGhLxCy9pWP6xkldVOKLrLKy';
-        $dynamic = Auth::passwordHash($secret);
+        /*
+        General tests, using pre-defined hashes generated by online tools
+        */
+
+        // Bcrypt - Version Y
```

Comment:
Does this test give us good coverage for all the new "Hash" classes?

Reply:
A combination of this and UsersAPI tests (to create&signIn) should give good enough coverage. We try defaults, as well as play around with options. There could be something platform-specific we would need to adjust to, but hashing algo, in general, should be well tested.